worlds/minecraft/data/excluded_locations.json

```diff
@@ -15,7 +15,12 @@
         "Sound of Music",
         "Star Trader",
         "When the Squad Hops into Town",
-        "With Our Powers Combined!"
+        "With Our Powers Combined!",
+	"Smells Interesting",
+	"Little Sniffs",
+	"Planting the Past",
+	"Smithing with Style",
+	"Careful Restoration"

```

Comment:
Not sure I would consider `smells interesting`, `little sniffs`, or `planting the past` as hard. Just a little time consuming. but I'm open to others thoughts on this.

Reply:
I chose that due to Sniffer Eggs having a low drop rate in one specific structure. Though since I was modifying drop rates for Smithing Templates anyway, I decided to raise the drop rates of them as well, which makes them being normal Advancements more reasonable.

Reply:
Yea, I can see your point. 
but also the underwater ruins are rather common and easy to find lots of if you just sail with a boat. So I'd say we make them maybe 3-5x more common from the sus gravel down there and knock them back to regular advancements. Thoughts?

Reply:
Sounds good to me! Will update with all of that soon.